src/OpenDB/src/swig/python/dbtypes.i

```diff
@@ -37,6 +37,98 @@
     $result = list;
 }
 
+%typemap(out) std::vector< T > {
```

Comment:
In opendb.i we have

%include <std_vector.i>

doesn't that take care of std::vector already?  Likewise string & pair

Reply:
For some reason when I removed the custom interface for std::pair it just returns a "SwigPyObject".. Is there an extra step to wrap the returned values? Also for std::vector, that was already there didn't really add it.